contracts/Example1.sol

```diff
@@ -1,7 +1,15 @@
 pragma solidity ^0.8.4;
 
 contract Example1 {
-    uint8 public day = 10;
-    uint256 public amount = 1000;
-    address public ownerAddr = 0x4e653b730367e303D30d8FB39d7350D07a60dfD6;
-}
\ No newline at end of file
+    uint8 public day;
+    uint16 public amount;
```

Comment:
uint type of amount should not be changed since the amount can easily overflow uint16. The expectation was to reorder and have this order which would require two slots

```
uint8 public day;
address public ownerAddr;
uint256 public amount;
```

Reply:
Got it, how much space does an address cost? Can uint8 and address fit into one slot?

Reply:
address is only 20 bytes or 160 bits